src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 600;
+    private static final int DEFAULT_RADIUS = 10;
+    private static ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
```suggestion
    private ColorSupplier colorSupplier = new ColorSupplier();
```
avoid static where it's not needed, static members are harder to mock and test